src/index.js

```diff
@@ -20,15 +24,68 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render({
+  display, result, opCode, handle,
+}) {
+  function calculate(i, j, opCode) {
+    if (opCode = '+') {
```

Comment:
```suggestion
    if (opCode === '+') {
```
"=" 은 값을 할당을 하고,
"===" 은 값을 비교하는 연산자 입니다!

"===" 와 "==" 의 차이는 알고 계시나요??

Reply:
오호이 조건문에서도 "="은 값을 할당하는 거군요!
아니요 차이 몰랐습니다. 감사합니다!!